forge/forge/op/eltwise_binary.py

```diff
@@ -441,3 +441,7 @@ def LogicalAnd(name: str, operandA: Tensor, operandB: Union[Tensor, Parameter])
     """
 
     return op("logical_and", name, operandA, operandA).get_tensor()
+
+
+def Mod(name: str, operandA: Tensor, operandB: Union[Tensor, Parameter]) -> Tensor:
```

Comment:
Can we call this op "Reminder" on Forge-FE as well?

Reply:
Updated @nvukobratTT